src/gc-stacks.c

```diff
@@ -226,6 +226,12 @@ void sweep_stack_pools(void)
                     t->stkbuf = NULL;
                     _jl_free_stack(ptls2, stkbuf, bufsz);
                 }
+#ifdef JL_TSAN_ENABLED
+                if (t->ctx.tsan_state) {
+                    t->ctx.tsan_state = NULL;
+                    __tsan_destroy_fiber(t->ctx.tsan_state);
```

Comment:
This is a NULL pointer?
```suggestion
                    __tsan_destroy_fiber(NULL);
```